src/main/java/com/bookshop/exception/CustomGlobalExceptionHandler.java

```diff
@@ -24,8 +24,7 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
             HttpHeaders headers,
             HttpStatusCode status,
             WebRequest request) {
-        Map<String, Object> body = new LinkedHashMap<>();
-        body.put("timestamp", LocalDateTime.now());
+        Map<String, Object> body = createBodyMapWithTime();
```

Comment:
Did you think about creating and using a response dto for this purpose instead of Map?

Reply:
Honestly, I didn't. What might be the advantages of this approach?

Reply:
In my opinion. with a 'dto' you will have fixed fields like timestamp, status, and errors. You can reuse it easily and increase code clarity